c7n/filters/policystatement.py

```diff
@@ -101,8 +101,9 @@ def process_resource(self, resource):
                 found = 0
                 for key, value in required_statement.items():
                     if key in ['Action', 'NotAction']:
-                        if key in statement and self.action_resource_case_insensitive(value) \
-                           == self.action_resource_case_insensitive(statement[key]):
+                        if key in statement and \
+                            set(self.action_resource_case_insensitive(value)) \
+                            == set(self.action_resource_case_insensitive(statement[key])):
```

Comment:
nit: I'm wondering if it makes sense to move the `set()` conversion to the [action_resource_case_insensitive() return](https://github.com/cloud-custodian/cloud-custodian/blob/feccb473e1b03872f570b1fd3434501fb6f506a2/c7n/filters/policystatement.py#L79) instead though. If we do that, it will fix the logic [here](https://github.com/cloud-custodian/cloud-custodian/blob/feccb473e1b03872f570b1fd3434501fb6f506a2/c7n/resources/ses.py#L140-L141) also.